app/services/applier/ashby_form_filler.rb

```diff
@@ -43,158 +38,5 @@ def hidden_element
     def response_field
       find(:css, "label[for='#{@locator}']").ancestor('fieldset', match: :first)
     end
-
```

Comment:
This is no longer required presumably as we have the actual payload? Is it all working?

Reply:
I just moved this from the children to the parent class, actually. Will implement a functional #click_submit_button method once we're finished with development testing.